mlcustomize/firstboot.ml

```diff
@@ -305,13 +305,19 @@ if not exist \"%%scripts_done%%\" (
 :: Pick the next script to run.
 for %%%%f in (\"%%scripts%%\"\\*.bat) do (
   echo running \"%%%%f\"
-  move \"%%%%f\" \"%%scripts_done%%\"
-  pushd \"%%scripts_done%%\"
+  pushd \"%%scripts%%\"
   call \"%%%%~nf\"
   set elvl=!errorlevel!
   echo .... exit code !elvl!
   popd
 
+  if !elvl! equ 0 (
+    echo Script succeeded, moving to scripts-done
+    move \"%%%%f\" \"%%scripts_done%%\"
+  ) else (
```

Comment:
As noted on https://github.com/libguestfs/virt-v2v/pull/94 there needs to be a special value here to indicate the retry behaviour, else we'll get boot loops.